test/stress.c

```diff
@@ -61,7 +61,7 @@ int read_from_fd()
     unsigned char c;
     int len = read(input_fd, &c, 1);
     if (len <= 0)
-	return EOF;
+    return EOF;
```

Comment:
TABs mean 8 space in the original code. I'm okay to reformat it.

But how do you reindent? Manually or automation by tools? This line should be 8 spaces instead of 4 spaces. Could you check again?